src/character.cpp

```diff
@@ -132,9 +132,20 @@ bool Character::move_effects()
             remove_effect("in_pit");
         }
     }
+    if (has_effect("grabbed")){
+        if ( get_dex() > get_str() ? dice(get_dex(), 7) : dice(get_str(), 7) < get_effect_dur("downed") ){
```

Comment:
Grabbing should not prevent the player from attacking, at that point you're
basically stunlocking the player, which is not ok.  It should either
prevent movement or turn move actions into an attempt to break the lock,
but most other actions should be allowed, though it would be ok to impede
them.